operator/pkg/controller/workspace/tfc_output.go

```diff
@@ -18,6 +21,92 @@ func (t *TerraformCloudClient) GetStateVersionDownloadURL(workspaceID string) (s
 	return stateVersion.DownloadURL, nil
 }
 
+func convertValueToString(val cty.Value) string {
+	ty := val.Type()
+	switch {
+	case ty.IsPrimitiveType():
+		switch ty {
+		case cty.String:
+			{
+				// Special behavior for JSON strings containing array or object
+				src := []byte(val.AsString())
+				ty, err := ctyjson.ImpliedType(src)
+				// check for the special case of "null", which decodes to nil,
+				// and just allow it to be printed out directly
+				if err == nil && !ty.IsPrimitiveType() && strings.TrimSpace(val.AsString()) != "null" {
```

Comment:
Is it worth adding a test for this null branch since we're dealing with a special case?